server/app.js

```diff
@@ -42,6 +42,20 @@ app.listen(3000, () => {
   `);
 });
 
+app.get('/v1/animals?name=', (req, res) => {
```

Comment:
this route should be located in `server/routes/wildlife.js` to stay consistent in our route strategy. We can fix this after approval, I don't think it'll break anything being here for now.